.make_ghpages/generate_json.py

```diff
@@ -10,6 +10,8 @@
 
 # Extract all the data. 
 
+ALLOWED_YAML_SUFFIXES = ['.yml', '.yaml']
```

Comment:
Aren't we shooting ourselves in the foot by allowing both suffixes in the registry?
We make our life harder both for users and maintainers.

I would propose to add a stupid pre-commit hook that raises whenever someone tries to commit a `.yaml` file, asking them to use `.yml` instead.

What do you think?

Reply:
I am fine with using one of those, but why did we chose `yml` and not `yaml`. According to the [yaml FAQ](https://yaml.org/faq.html) `yaml` is the recommended way.

Reply:
Thanks, I wasn't aware. `yaml` it is then!

Reply:
I renamed all the configuration files.